streams-bootstrap-core/src/main/java/com/bakdata/kafka/streams/StreamsCleanUpConfiguration.java

```diff
@@ -84,4 +86,30 @@ void runResetHooks() {
     void runTopicDeletionHooks(final String topic) {
         this.topicHooks.forEach(hook -> hook.deleted(topic));
     }
+
+    /**
+     * Converts this configuration to a {@link ConsumerCleanUpConfiguration}, transferring
+     * registered clean and reset hooks.
+     *
+     * @return {@link ConsumerCleanUpConfiguration} instance
+     */
+    public ConsumerCleanUpConfiguration toConsumerCleanUpConfiguration() {
+        final ConsumerCleanUpConfiguration configuration = new ConsumerCleanUpConfiguration();
+        this.cleanHooks.forEach(configuration::registerCleanHook);
+        this.resetHooks.forEach(configuration::registerResetHook);
+        return configuration;
+    }
+
+    /**
+     * Converts this configuration to a {@link ProducerCleanUpConfiguration}, transferring
+     * registered clean and topic hooks.
+     *
+     * @return {@link ProducerCleanUpConfiguration} instance
+     */
+    public ProducerCleanUpConfiguration toProducerCleanUpConfiguration() {
+        final ProducerCleanUpConfiguration configuration = new ProducerCleanUpConfiguration();
+        this.cleanHooks.forEach(configuration::registerCleanHook);
+        this.topicHooks.forEach(configuration::registerTopicHook);
+        return configuration;
+    }
```

Comment:
These two methods seem to be unused. Does it make sense to keep them around regardless?